src/shared/platform/lind_platform.c

```diff
@@ -765,18 +765,25 @@ int lind_flock (int fd, int operation)
     LIND_API_PART3;
 }
 
-int lind_pipe(int* pipefds){
+int lind_pipe(int* pipefds, int cageid){
     LIND_API_PART1;
-    callArgs = Py_BuildValue("(i[])", LIND_safe_sys_pipe);
+    callArgs = Py_BuildValue("(i[i])", LIND_safe_fs_pipe, cageid);
     LIND_API_PART2;
     COPY_DATA(pipefds, 2*sizeof(int))
     LIND_API_PART3;
 }
 
 int lind_pipe2(int* pipefds, int flags){
     LIND_API_PART1;
-    callArgs = Py_BuildValue("(i[i])", LIND_safe_sys_pipe2, flags);
+    callArgs = Py_BuildValue("(i[i])", LIND_safe_fs_pipe2, flags);
     LIND_API_PART2;
     COPY_DATA(pipefds, 2*sizeof(int))
     LIND_API_PART3;
 }
```

Comment:
We should clarify that pipe2 is not implemented, and anyway it should have cageid here too.